manifests/baremetal/coredns-corefile.tmpl

```diff
@@ -1,6 +1,6 @@
 . {
     errors
-    health
+    health :18080
```

Comment:
I don't think we should change baremetal's code, lets just alert them of the issue and let them do it on their own.

Reply:
agreed

Reply:
yes, sorry :(